l5kit/l5kit/evaluation/extract_metric.py

```diff
@@ -0,0 +1,59 @@
+import numpy as np
+
+from . import read_gt_csv, read_pred_csv
+from .metrics import neg_multi_log_likelihood
+
+
+def compute_error_csv(ground_truth_path: str, inference_output_path: str) -> np.ndarray:
+    """
+    Arguments:
+        ground_truth_path (str): Path to the ground truth csv file.
+        inference_output_path (str): Path to the csv file containing network output.
+    """
+    ground_truth = {}
+    inference = {}
+
+    for el in read_gt_csv(ground_truth_path):
+        ground_truth[el["track_id"] + el["timestamp"]] = el
```

Comment:
Are track_id and timestamp int or strings? Anyways, in both cases, is it guaranteed that the addition result is unique?

Reply:
They are strings as they are read from a csv (and not converted)
the combination is unique in our dataset. Even if it's not for prediction (tampering) validate will fail as the number of rows will differ